Tests/SonarRunner.Shim.Tests/PropertiesFileGeneratorTests.cs

```diff
@@ -261,11 +261,58 @@ public void FileGen_AdditionalProperties()
             AssertPropertiesFilesCreated(result, logger);
 
             SQPropertiesFileReader provider = new SQPropertiesFileReader(result.FullPropertiesFilePath);
-            AssertExpectedConfigSetting("key1", "value1", provider);
-            AssertExpectedConfigSetting("key.2", "value two", provider);
-            AssertExpectedConfigSetting("key.3", " ", provider);
+            provider.AssertSettingExists("key1", "value1");
+            provider.AssertSettingExists("key.2", "value two");
+            provider.AssertSettingExists("key.3", " ");
 
-            AssertSettingDoesNotExist("server.key", provider);
```

Comment:
I renamed and moved the assertions to the SQPropertiesFileReader (previously existed in two test classes).